generated/nidcpower/nidcpower/_converters.py

```diff
@@ -275,3 +275,25 @@ def convert_chained_repeated_capability_to_parts(chained_repeated_capability):
     return [','.join(collections.OrderedDict.fromkeys(x)) for x in repeated_capability_lists]
 
 
+# Getter converter for _IsolationState
+___isolation_state_enum_value_to_converted_value_dict = {
+    1128: True,
+    1129: False,
+}
+
+
+def convert_from___isolation_state_enum_value(enum_value):
+    return ___isolation_state_enum_value_to_converted_value_dict[enum_value]
+
+
+# Setter converter for _IsolationState
+___isolation_state_converted_value_to_enum_value_dict = {
```

Comment:
As far as I'm aware, there are ways to reverse a dict in Python, e.g. https://stackoverflow.com/a/61102213, but these methods still require a separate variable to be declared. To avoid declaring an additional dict, we can either reverse the dict on-the-fly inside the converter function, or loop through the first dict to search for the corresponding key for a value (e.g. https://stackoverflow.com/a/2569076) inside the converter function. Both of these approaches seem to be more less efficient in terms of execution time compared to declaring an additional dict outside the converter function if this converter function need to be called repeatedly. 

@marcoskirsch Should I go with one of these approaches?

Reply:
After thinking about it, I think the approach I favor the most is to codegen inside the converter function.

```
    return {
        1 : RED,
        2 : BLUE,
        3: GREEN,
    }[value]
```

I think this is cleanest, polutes namespaces least, and is more Pythonic.

Reply:
I have updated the converter codegen logic to codegen the dicts inside the converter functions directly in my latest commit.